src/database/migrations/20231220235718_removeParticipantIdColumnFromAudit.ts

```diff
@@ -0,0 +1,24 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(
```

Comment:
I think `knex.raw` is unavoidable here, but perhaps we could leverage a bit more `knex` syntax here. Perhaps something like:
```
await knex('auditTrails')
  .whereNotNull('participantId')
  .update({
    eventData: knex.raw("JSON_MODIFY(eventData, '$.participantId', ?)", [participantId])
  });
```
Not a big deal if it's not an easy change though.

Reply:
Managed to change one. The other one I couldn't work out how to get the where statement to work.

Reply:
I think you'd need to use `knex.raw `in the `where` clause itself. Maybe the following?
```
await knex('auditTrails')
  .whereNotNull(knex.raw("JSON_VALUE(eventData, '$.participantId')"))
  .update({
    participantId: knex.raw("JSON_VALUE(eventData, '$.participantId')")
  });
```

Reply:
![VKextfOyrO](https://github.com/IABTechLab/uid2-self-serve-portal/assets/152250444/9bd5817b-222a-4640-b40c-e44d38a8aae6)
Yeah I tried that, it throws this error. It seems to work with just where but not whereNotNull.

Reply:
Ah right you are. What about:
```
await knex('auditTrails')
  .where(knex.raw("JSON_VALUE(eventData, '$.participantId') IS NOT NULL"))
  .update({
    participantId: knex.raw("JSON_VALUE(eventData, '$.participantId')"),
  });
```